src/constants/urls.ts

```diff
@@ -1,3 +1,6 @@
-export const RDS_API_BASE_URL = "https://api.realdevsquad.com";
+export const RDS_BASE_API_URL = "https://api.realdevsquad.com";
+export const RDS_BASE_STAGING_API_URL = "https://staging-api.realdevsquad.com";
+export const RDS_BASE_DEVELOPMENT_API_URL = "http://localhost:3000"; // If needed, modify the URL to your local API server
```

Comment:
This API URL can be changed to any local development URL for using the backend. Maybe we will be add more environments later for testing, local etc.